mappers/bluetooth_mapper/README.md

```diff
@@ -0,0 +1,318 @@
+# Bluetooth Mapper
```

Comment:
I don't think it's suitable to copy design document here. If we want people to read the design, can provide a link.

In README, just need to provide usage, input & output etc.

Reply:
Done, thanks